src/artofillusion/polymesh/MeshUnfolder.java

```diff
@@ -138,449 +92,33 @@
 	 *                assigned a vertex index of -1. They won't be displayed
 	 *                by the UVMappingCanvas widget.
 	 */
-	public MeshUnfolder(FacetedMesh mesh, TriangleMesh trimesh,
-			int[] vertexTable, int[] faceTable) {
+	public MeshUnfolder(FacetedMesh mesh, TriangleMesh trimesh, int[] vertexTable, int[] faceTable) {
 		this.mesh = mesh;
 		this.trimesh = trimesh;
 		this.vertexTable = vertexTable;
 		this.faceTable = faceTable;
 	}
-	
+
 	public boolean unfold(BTextArea textArea, double res) {
-		//return unfoldAbf(textArea, res);
-		return unfoldLinearAbf(textArea);
+          return unfoldLinearAbf(textArea);
 	}
 
-	/**
-	 * Unfolds the mesh using the ABF++ algorithm
-	 * 
-	 * @param res
-	 *                The residual to reach
-	 * @param textArea
-	 *                The text area to print results to
-	 * 
-	 */
-	public boolean unfoldAbf(BTextArea textArea, double res) {
-		// dump mesh;
-		TriangleMesh.Edge[] edges = trimesh.getEdges();
-		int nedges = edges.length;
-		long totaltime;
-		QMR qmr = null;
-		int ind[];
-		int jnd[];
-		double[] idata;
-		double[] jdata;
-		SparseVector JstarVecI;
-		SparseVector JstarVecJ;
-		SparseVector JstarstarVec;
-		double value;
-		int iter = 0;
```

Comment:
Are we sure that this is inoperable, or is it just that the UI doesn't make it visible? If the second, maybe we should hook it up and see how well it works...

Reply:
I see through plugin history. Initially was implemented abf++ algoritm. Than added linear abf implementation and only it used since 2008 year